package.json

```diff
@@ -5,6 +5,8 @@
   "version": "0.1.0",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^1.9.0",
+    "@typescript-eslint/parser": "^1.9.0",
```

Comment:
I think these two typescript packages are also supposed to be in dev dependencies. Forgot to include these in the last comment.